keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md

```diff
@@ -164,6 +164,13 @@ minReadySeconds is already available as an optional field for Deployments,
 DaemonSets, ReplicasSets and Replication Controllers. Enabling this option 
 helps in bringing StatefulSets on par with other workload controllers.
 
+Note: The important point to understand is when will a pod be considered
+Ready which might depend on the container probes configured. For example,
+for a pod with single container, if there is a readiness check with `initialDelaySeconds`
+configured for the container in the pod for the pod to considered
```

Comment:
"for the container in the pod for the pod to considered" .. could we make this clearer?